proto/anki/collection.proto

```diff
@@ -107,5 +116,6 @@ message Progress {
     FullSync full_sync = 4;
     NormalSync normal_sync = 5;
     DatabaseCheck database_check = 6;
+    string media_processing = 8;
```

Comment:
This again. 😞

Reply:
?

If you're referring to the blank=null discussion from above, oneofs are a different story (and the new optional functionality is actually handled internally like a hidden oneof)

Reply:
Sorry, that was rather vague. 😅
The build fails if I write `= 7`. This time I can only get around this by skipping this number, but without a doubt it's the same issue as in https://github.com/ankitects/anki/pull/1677#discussion_r811070799.

Reply:
Ah, I get you now :-) I tried to reproduce it on a Windows machine, and got an error about a missing backend_pb2.py file or similar one time, but then the build completed fine the second time. Do you use --disk_cache in your user.bazelrc? I don't use it on Windows, and it's possible that there's a race condition in the build process on Windows leading to a bad output, which then gets cached on your system. If that's the case, you probably don't need to change the number, and a minor change like a comment or extra newline would probably fix it too.

Reply:
> got an error about a missing backend_pb2.py file or similar one time

I got used to these, as restarting once or twice always resolves them.

> Do you use --disk_cache in your user.bazelrc? I don't use it on Windows, and it's possible that there's a race condition in the build process on Windows leading to a bad output, which then gets cached on your system.

You're right! Deleting that cache fixed it. I would have tried that sooner, but I thought `bazel clean --expunge` would have the same effect.

> If that's the case, you probably don't need to change the number, and a minor change like a comment or extra newline would probably fix it too.

I'll try that the next time. I understand this means it was just chance that tweaking the numbers worked, and I actually only needed to keep making changes until a build succeeded at first pass (without the usual error mentioned at the top).

Reply:
If the backend_pb2.py thing keeps coming up, you might want to try commenting `stderr=subprocess.DEVNULL` in protoc_wrapper.py to see if that gives us any hint as to why the file is missing - it just makes the build a bit noisier. It might also give us a hint as to what exactly is going wrong in the bad cache case.